keys.hh

```diff
@@ -596,8 +597,8 @@ public:
 
         bool operator()(const TopLevel& k1, const TopLevel& k2) const {
             return prefix_equality_tri_compare(prefix_type->types().begin(),
-                prefix_type->begin(k1), prefix_type->end(k1),
-                prefix_type->begin(k2), prefix_type->end(k2),
+                prefix_type->begin(k1.representation()), prefix_type->end(k1.representation()),
```

Comment:
Which part do you mean? `prefix_equality_tri_compare` calls `tri_compare`, which calls `abstract_type::compare`, which is indeed linearizing. So is `abstract_type::hash`. But Avi said above to leave `compare` and `hash` for a another series, because this one is big enough.

Reply:
I meant `representation()`, or does that return a `managed_bytes_view` now?

Reply:
It returns `managed_bytes_view` now.

Reply:
Oh, then my comment is void.